### ISSUE TITLE
Clarification on useCallback function name on beta docs

### ISSUE THREAD

**Issue by kkyusufk** [reactjs/reactjs.org#5079]
In the react beta docs,  `useCallback` api reference page gives the following example
<img width="925" alt="image" src="https://user-images.githubusercontent.com/22116082/190866735-3579d750-9d08-46ef-9ad0-113deda4643f.png">

I think the constant naming `memoizedFn` leads to some confusion on what `useCallback` does. 
People tend to assume that `useCallback` actually memoizes the function but it does not do that. It returns the same instance of the function between re-renders.
An appropriate name of that variable should be `originalFn`. 
I maybe wrong so please let me know if the above makes sense. 

Big fan of the beta docs btw ❤️ , beautiful documentation and excellent examples!

**Comment by gaearon** [comment]
I agree with the concern but I don’t think your suggestion is much clearer.

**Comment by kkyusufk** [comment]
how about a simple `returnedFn`? Since `useCallback` does return a function.
Maybe I can send a small PR and we can start a discussion on it. More folks can chime in.

**Comment by gaearon** [comment]
I don't think that's better either since you could prepend any name with `returned` and it wouldn't change the meaning.

**Comment by lxmarinkovic** [comment]
I agree with @gaearon; it would change the meaning
